docker-compose.yml

```diff
@@ -48,14 +48,16 @@ services:
 
     downtime:
         image: observatorycontrolsystem/downtime:2.3.2
+        logging: 
+            driver: none
```

Comment:
what this line for / is it different then the default?

Reply:
Oops, I was just trying to supress the downtime output during development. Removed.